server/core/region.go

```diff
@@ -58,6 +60,7 @@ type RegionInfo struct {
 	replicationStatus *replication_modepb.RegionReplicationStatus
 	QueryStats        *pdpb.QueryStats
 	flowRoundDivisor  uint64
+	buckets           unsafe.Pointer
```

Comment:
Why not define as `*metapb.Buckets` directly?

Reply:
It may be updated by concurrency。

Reply:
We better add a comment to explain the concurrency case here.

Reply:
fix